contracts/tools/DepositHelper.sol

```diff
@@ -199,6 +193,44 @@ contract DepositHelper is ReentrancyGuard {
     }
   }
 
+  function convertAndIncreaseAmount(
+    bytes memory asset0SwapData,
+    bytes memory asset1SwapData,
+    address tokenIn,
+    uint amountIn,
+    IVeTetu ve,
+    uint tokenId
+  ) external nonReentrant returns (
+    uint lockedAmount,
+    uint power,
+    uint unlockDate
+  ) {
+    uint bptBalance = convertToTetuBalancerPoolToken(
+      asset0SwapData,
+      asset1SwapData,
+      tokenIn,
+      amountIn,
+      BALANCER_VAULT,
+      BALANCER_POOL_ID,
+      BALANCER_POOL_TOKEN,
+      ASSET0,
+      ASSET1
+    );
+    _approveIfNeeds(BALANCER_POOL_TOKEN, bptBalance, address(ve));
+
+    ve.increaseAmount(BALANCER_POOL_TOKEN, tokenId, bptBalance);
+
+    lockedAmount = ve.lockedAmounts(tokenId, BALANCER_POOL_TOKEN);
+    power = ve.balanceOfNFT(tokenId);
+    unlockDate = ve.lockedEnd(tokenId);
+
+    // send back possible tokens
+    uint balance = IERC20(BALANCER_POOL_TOKEN).balanceOf(address(this));
```

Comment:
add sending BPT to `convertAndCreateLock` too
the contract immutable, if something goes wrong we need to be sure the asset are not stuck forever

Reply:
I think I can not do it, because I need first increase amount
```
    ve.increaseAmount(BALANCER_POOL_TOKEN, tokenId, bptBalance);
```